MapleServer2/PacketHandlers/Game/ClubHandler.cs

```diff
@@ -17,7 +18,9 @@ public class ClubHandler : GamePacketHandler
         private enum ClubMode : byte
         {
             Create = 0x1,
-            Join = 0x3,
+            Join = 0x3, //for creating
```

Comment:
Not sure I understand this comment, is it needed?

Reply:
I suppose it's not needed. It was real odd having an invite and join enum so I just left it for clarification.